### ISSUE TITLE
Dividing input changes BatchNorm output

### ISSUE THREAD

**Issue by vuvko** [apache/incubator-mxnet#7909]
I was trying to experiment with ResNet-152 model downloaded from [Model Zoo](https://mxnet.incubator.apache.org/model_zoo/). For some reason first batch normalization vary it's output if I divide (or apply any other elementwise operation to) it's input.

## Environment info
Operating System: ArchLinux 
`
Linux archshad 4.12.10-1-ARCH #1 SMP PREEMPT Wed Aug 30 12:18:42 CEST 2017 x86_64 GNU/Linux
`

Compiler: 

- `gcc (GCC) 7.2.0`
- `gcc-5 (GCC) 5.4.0`
- `Cuda compilation tools, release 8.0, V8.0.61`

Package used (Python/R/Scala/Julia): Python

MXNet version: 0.11.0 (behavior was the same on 0.10.0)

Python version: `Python 3.6.2`, `Python 2.7.13`

## Minimum reproducible example
I wrote [script](https://gist.github.com/vuvko/f46eac67a285f6f5040e630a87e5cd99) for checking outputs of model on test image.

## Steps to reproduce
Run code from gist with some colored image saved resnet-152 model.

## What have you tried to solve it?

Changing library versions, python version

**Comment by vuvko** [comment]
I don't think it's desired behavior with parameter `use_global_stats` set to `False`. Am I wrong somewhere?

**Comment by vuvko** [comment]
Also if this can help, I used [this](https://i.imgur.com/XJOBiiO.jpg) image for testing.

**Comment by vuvko** [comment]
Ok, so the problem I loading model for testing and in that case parameter `use_global_stats` is just ignored (as set to `True`)
